openbas-front/src/admin/components/common/injects/UpdateInject.tsx

```diff
@@ -81,7 +79,6 @@ const UpdateInject: React.FC<Props> = ({ open, handleClose, onUpdateInject, mass
             handleClose={handleClose}
             onUpdateInject={onUpdateInject}
             isAtomic={isAtomic}
-            teamsFromExerciseOrScenario={teamsFromExerciseOrScenario}
             {...props}
```

Comment:
When I try to update an atomicTesting, I have any teams on his definition: 
![image](https://github.com/user-attachments/assets/908a8dbd-300c-48e0-b810-f12f3b3ab168)